pythonmarketo/client.py

```diff
@@ -236,7 +237,29 @@ def create_lead(self, lookupField, lookupValue, values):
             ]
         }
         return self.post(data)
+    
+    def run_request_campaign(self, campaignID, leadsID, values):
+        token_list = [{'name':'{{' + k + '}}', 'value':v} for k, v in values.items()]
+        leads_list = [{'id':items} for items in leads_list] 
+        data={
+          'input': {"leads":
+                    leads_list                  
+                   ,"tokens":
```

Comment:
Updated...
- ThanksSandip
 


     On Friday, May 15, 2015 1:52 PM, Arunim Samat <notifications@github.com> wrote:
   

 In pythonmarketo/client.py:> @@ -236,7 +237,29 @@ def create_lead(self, lookupField, lookupValue, values):
>              ]
>          }
>          return self.post(data)
> +    
> +    def run_request_campaign(self, campaignID, leadsID, values):
> +        token_list = [{'name':'{{' + k + '}}', 'value':v} for k, v in values.items()]
> +        leads_list = [{'id':items} for items in leads_list] 
> +        data={
> +          'input': {"leads":
> +                    leads_list                  
> +                   ,"tokens":
Minor_Fix:leads_list, "," should be in the same line as leads_list—
Reply to this email directly or view it on GitHub.